--- driver.org/frontcontroller/proton/proton.c.old	2019-03-02 23:13:37.091076106 +0300
+++ driver/frontcontroller/proton/proton.c	2019-03-02 23:21:52.394915397 +0300
@@ -102,7 +102,7 @@
 #define FRONTPANEL_MINOR_RC             1
 #define LASTMINOR                 	    2
 //#define ENABLE_SCROLL
-//#define ENABLE_CLOCK_SECTION
+#define ENABLE_CLOCK_SECTION
 
 static tFrontPanelOpen FrontPanelOpen [LASTMINOR];
 
